src/main/java/com/internet/mixshop/model/Role.java

```diff
@@ -0,0 +1,34 @@
+package com.internet.mixshop.model;
+
+public class Role {
+    private RoleName roleName;
+    private Long id;
+
+    public Role(RoleName roleName) {
```

Comment:
let's make it private since we have method of()

Reply:
done